contracts/cw-staking/src/lib.rs

```diff
@@ -16,41 +16,46 @@ pub mod host_staking {
     pub use super::providers::osmosis::Osmosis;
 }
 
-#[cfg(feature = "boot")]
-pub mod boot {
+#[cfg(feature = "cw-orch")]
```

Comment:
I prefer to use "interface" as the feature name because it's more informative when using it.